### PR TITLE
while adding panel it takes to edit panel page

### LINKED ISSUE #443: [BUG] Dashboard add panel workflow
# Description
When you add a dashboard panel, it should take you to the page to edit that panel

# Steps to Reproduce
1. Make a dashboard
2. In the top right, click the `+ Add Panel` button

# Expected Behavior
Currently this adds the panel but then you have to click the panel and click Edit to start editing it. Instead, once you click the `+ Add Panel` button, it should make the panel and immediately take you to the page where you edit that panel

# Environment
- Commit Hash: eabb398

# (Optional) Additional Context
Add anything else we should know about this problem.

### PR DESCRIPTION
**Pull request by PratikshaPatil77**
# Description
When you add a dashboard panel, it takes you to the page to edit that panel. Once you click the + Add Panel button, it make the panel and immediately take you to the page where you can edit that panel.

Fixes #443 

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ x ] I have self-reviewed this PR.
- [ x ] I have removed all print-debugging and commented-out code that should not be merged.
- [ x ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

**Review by AndrewHess — COMMENTED** [review]
When I add a panel, add a query, select the Data Source, select a chart type and click Apply, I don't see the new panel on the dashboard. Is this working for you?

**Comment by PratikshaPatil77** [comment]
> When I add a panel, add a query, select the Data Source, select a chart type and click Apply, I don't see the new panel on the dashboard. Is this working for you?

I apologize . Initially, I only verified the redirection to the edit panel page after clicking "Add Panel."
I overlooked confirming the panel's successful saving. Checking it now.

**Comment by PratikshaPatil77** [comment]
Changes will be ready soon.

**Comment by PratikshaPatil77** [comment]
@AndrewHess  @nbhavana @sonamgupta21 
Changes are ready for review.
 Please have a look and if any changes required please let me know.

File: static/js/dashboard.js [hunk 1]
```diff
@@ -1053,7 +1052,13 @@ function addPanel(panelToDuplicate) {
             },
             "queryType": "",
         });
-
+        editPanelInit(panelIndex);
+        $('.panelEditor-container').show();
+        $('#app-container').hide();
+        $('.panelDisplay #panelLogResultsGrid').empty();
+        $('.panelDisplay .big-number-display-container').hide();
+        $('.panelDisplay #empty-response').hide();
```
Comment (AndrewHess) [inline static/js/dashboard.js:1060]: nit: it looks like this is indented one indent too far, right?

**Review by AndrewHess — COMMENTED** [review]
Looks pretty good to me, just a minor comment about indentation

**Review by sonamgupta21 — CHANGES_REQUESTED** [review]
There's one issue: When we duplicate a panel, it opens the edit panel screen, which shouldn't happen.

**Review by sonamgupta21 — APPROVED** [review]

**Pull request merged**